react-task/src/components/Input.jsx

```diff
@@ -1,29 +1,57 @@
 import React from "react";
 import PropTypes from "prop-types";
 
-const Input = ({ type, id, placeholder, helpText }) => (
-  <div class="form-floating mb-3">
-    <input
-      type={type}
-      class="form-control"
-      id={id}
-      placeholder={placeholder}
-      aria-describedby={helpText && "help"}
-    ></input>
-    <label for={id}>{placeholder}</label>
-    {helpText && (
-      <div id="help" class="form-text">
-        {helpText}
-      </div>
-    )}
-  </div>
-);
+const Input = ({
+  type,
+  id,
+  placeholder,
+  helpText,
+  value,
+  handleDataChange,
+  warningText,
+}) => {
+  const handleOnChange = (event) => {
```

Comment:
The general standard we follow to handle these sorts of functions will be `handle<event>`

```suggestion
  const handleChange = (event) => {
```